### ISSUE TITLE
HuExpress command line POSIX compilant fails

### ISSUE THREAD

**Issue by mckaygerhard** [kallisti5/huexpress#13]
The emulator always open the gui, or the hide gui feature are not documented! this its need for guiless implementation on others systems just like frontends or test for management..

currently as hugo, does not support parsin of arguments in command line as normal unix-like or posix OS's, the rom file must be parse firts.

the problem are clarelly see in iniconfig.c#L378 that assumed that file rom always are firs..

the other must that no extension must be requested.. this are a no necesary due the offset of the file defined the contents of the card game rom.

**Comment by kallisti5** [comment]
yeah, the command line arguments in HuGO were the same.  Definitely something that needs cleaned up.
